src/pages/filter/ui/filterAttributes.store.ts

```diff
@@ -25,6 +27,9 @@ export class FilterAttributesStore {
   private datasetStore: DatasetStore
   private filterStore: FilterStore
 
+  isAllMode = false
```

Comment:
better to create a property 'currentMode: ModeTypes' therefore you won't need to keep a weird boolean prop.
I could be wrong, pls check it